packages/manager/src/MainContent.tsx

```diff
@@ -333,14 +333,14 @@ export const MainContent = () => {
                       margin: '0 auto',
                       maxWidth: `${theme.breakpoints.values.lg}px !important`,
                       pt: {
-                        md: 1.5,
+                        md: theme.spacing(3),
```

Comment:
I'm seeing new eslint rules that say theme.spacing() is deprecated. While we're at it, maybe we can switch to the new rules:
![image](https://github.com/user-attachments/assets/279737af-88fd-4080-97b5-cbefb4ebacfc)
```suggestion
                        md: theme.spacingFunction(24),
```